src/js/components/CourseItem/CourseItem.js

```diff
@@ -8,44 +8,50 @@ import Button from '../Button';
 import styles from './styles';
 
 const CourseItem = ({
-  title, timing, date, text, classes, isAuth
-}) => (
-  <Paper className={classes.root}>
-    <div className={first}>
-      <h5 className={listTitle}>
-        {title}
-      </h5>
-      <div className={group}>
-        <span className={timeLabel}>
-          {timing}
-        </span>
-        <span className={dateLabel}>
-          {date}
+  title, timing, date, text, classes, isAuth, item, ...props
+}) => {
+  const deleteCourse = () => props.handleDeleteCourse(item);
+
+  return (
+    <Paper className={classes.root}>
+      <div className={first}>
+        <h5 className={listTitle}>
+          {title}
+        </h5>
+        <div className={group}>
+          <span className={timeLabel}>
+            {timing}
+          </span>
+          <span className={dateLabel}>
+            {date}
+          </span>
+          {isAuth && (
```

Comment:
for now the better way  to avoid isAuth check for some buttons is to crecte some element which knows if user already authenticated, with context it's looks easy for implement btw